Source/Charts/Data/Implementations/Standard/ChartData.swift

```diff
@@ -373,35 +204,27 @@ open class ChartData: NSObject, ExpressibleByArrayLiteral
     /// - parameter label:
     /// - parameter ignorecase:
     /// - returns: The DataSet Object with the given label. Sensitive or not.
-    @objc open func getDataSetByLabel(_ label: String, ignorecase: Bool) -> Element?
+    @objc open func dataSet(forLabel label: String, ignorecase: Bool) -> Element?
     {
         guard let index = index(forLabel: label, ignoreCase: ignorecase) else { return nil }
         return self[index]
     }
     
-    @objc open func getDataSetByIndex(_ index: Index) -> Element!
+    @objc(dataSetAtIndex:)
+    open func dataSet(at index: Index) -> Element?
     {
-        if index < 0 || index >= _dataSets.count
-        {
-            return nil
-        }
-        
+        guard dataSets.indices.contains(index) else { return nil }
```

Comment:
the [imp](https://github.com/apple/swift/blob/143c39bd7c3972f3416b83ac9366c93135319cf7/stdlib/public/core/SequenceAlgorithms.swift#L510) for `contains` is kind of confusing if it's O(1) or O(n), can you help check?
It seems `indices` here is type of `CountableRange<Int>`, however I cannot find `CountableRange` class nor extension, seems removed? There is also imp like [this](https://github.com/apple/swift/blob/e485c1506b2c37403fd0004d4e65914fc4f42309/stdlib/public/core/Range.swift#L166)
However from the func description, it is refering to  `SequenceAlgorithms.swift`

Reply:
It is O(1). `CountableRange` is now just a (deprecated) type alias:
```
@available(*, deprecated, renamed: "Range")
public typealias CountableRange<Bound: Strideable> = Range<Bound>
  where Bound.Stride : SignedInteger
```

The implementation of `Range<Bound>` is:
```
  @_inlineable
  public func contains(_ element: Bound) -> Bool {
    return lowerBound <= element && element < upperBound
  }
```